### DISCUSSION TITLE
What can I do to make hardhat display the full stack information?

### DISCUSSION THREAD

**Discussion by ziyiyu** [NomicFoundation/hardhat#4116] [Troubleshooting]
When I run the test script on the hardhat local host, I want to get ERROR StackTrace. If I deploy a contract through hardhat, I can get useful information. But if the contract already exists, it will display as follows. Even though I have the corresponding contract code, I can't tell where the error occurred. What can I do to make hardhat display the full stack information?
![image](https://github.com/NomicFoundation/hardhat/assets/26920829/6b82199c-34c9-47c0-9304-47c0dd882b41)

**Comment by fvictorio** [comment]
If that address corresponds to a contract from your project, then you should see it there. If you aren't, that might be a bug on our side.

If you can create a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of this, please open an issue with it.
